OCAT/client/resources/js/components/shared/services/assessment.service.js

```diff
@@ -14,12 +14,7 @@ export class AssessmentService {
 
```

Comment:
The description of the commit, accidently has the name 'tomcat' and it. Please ignore that word. 
My orinial commit message was: 'Simpliefied or corrected the logic got the ocat server call for assessments.'
To current my spelling fast - I was pasting the comment into google search bar and clicking the suggested corrected version of my search - well apparently that not only corrected my typos but also made some logical corrections. I learned that this is not a good approach for correcting typos.
![image](https://user-images.githubusercontent.com/54749949/111636619-095cf880-87cf-11eb-9907-19866f7457e3.png)